### PR TITLE
add extract to postcss

### PR DESCRIPTION
**Pull request by jamesBennett**
Generate CSS file from component internal. 
copy CSS that's would be injected in header and write that to a styles.css file in cjs and esm

### CODE REVIEW

**Review by cachilders — DISMISSED** [review]

File: packages/Accordion/package.json [hunk 1]
```diff
@@ -5,6 +5,10 @@
     "main": "dist/cjs/Accordion.js",
     "module": "dist/esm/Accordion.js",
     "types": "dist/cjs/index.d.ts",
+    "files": [
+        "dist/cjs/styles.css",
+        "dist/cjs/styles.css"
```
Comment (cachilders) [inline packages/Accordion/package.json:10]: Twice?

Comment (jamesBennett) [reply packages/Accordion/package.json:10]: oops that's suppose to be "dist/esm/styles.scs" for the second one. let me fix that

Comment (jamesBennett) [reply packages/Accordion/package.json]: updated

**Review by cachilders — APPROVED** [review]
nice

**Pull request merged**